how-to-ssr/public/javascripts/oc-client.js

```diff
@@ -0,0 +1,509 @@
+'use strict';
+/* eslint no-var: 'off' */
+/* eslint prefer-arrow-callback: 'off' */
+
+var oc = oc || {};
+
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    // AMD. Register as an anonymous module
+    define(['exports', 'jquery'], function(exports, $) {
+      root.oc = $.extend(exports, root.oc);
```

Comment:
@matteofigus as per your comment I think that this is what we need now.

// cc @navamgupta